builtin/providers/aws/resource_aws_security_group_rule_test.go

```diff
@@ -545,6 +582,38 @@ resource "aws_security_group_rule" "ingress_1" {
 }
 `
 
+const testAccAWSSecurityGroupRuleIngress_protocolConfig = `
+resource "aws_vpc" "tftest" {
+  cidr_block = "10.0.0.0/16"
+
+  tags {
+    Name = "tf-testing"
+  }
+}
+
+resource "aws_security_group" "web" {
+  name        = "terraform_acceptance_test_example"
+  description = "Used in the terraform acceptance tests"
```

Comment:
I think it's better to omit these to protect us against future collision - let the tags stick around for taxonomy purposes.